### ISSUE TITLE
Requesting Assistance: Export Posts as Individual Files

### ISSUE THREAD

**Issue by joshvickerson** [anchorcms/anchor-cms#955]
I also posted this in the [Anchor Forums](http://forums.anchorcms.com/help/dump-all-posts-to-individual-files) but they don't seem to be very active these days so I'm also posting here.

I'm hoping someone can help me put together a script to dump all posts in anchor to individual files, either as markdown or HTML.

I'm not really sure where to start and would like some help.

**Comment by TheBrenny** [comment]
Hey! I wouldn't mind helping. Shoot me an email whenever, you can find it on my GitHub profile.

I imagine that this just involves reading from the database and exporting the data to files, or as a file that's downloadable.

**Comment by joshvickerson** [comment]
Whoops! Forgot I opened this issue. I ended up figuring it out using MySQL's outfile command (which I didn't know existed). I modified the suggested answer from [this StackOverflow question](http://stackoverflow.com/questions/23389518/export-mysql-rows-into-different-files) to export every post into its own html file. It worked reasonably well but was a tiny bit of a pain.

Ultimately what I'd like to do is make a plugin to do this, but we're a long way from plugins as far as I know.
